kratos/modeler/cad_io_modeler.h

```diff
@@ -0,0 +1,145 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//
```

Comment:
You should add your name as author in the files

Reply:
I use to avoid that due to bad experience ;)